lib/loadGltfUris.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var fs = require('fs');
+var path = require('path');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+var isDataUri = require('./isDataUri');
+var dataUriToBuffer = require('data-uri-to-buffer');
+var async = require('async');
+
+module.exports = loadGltfUris;
+
+function loadGltfUris(gltf, basePath, callback) {
+    var shaders = gltf.shaders;
+
+    //Iterate through each shader and load its uri
+    if (defined(shaders)) {
+        var shaderIds = Object.keys(shaders);
+        async.each(shaderIds, function(shaderId, asyncCallback) {
+            var shader = shaders[shaderId];
+            var uri = shader.uri;
+            shader.extras = defaultValue(shader.extras, {});
+            loadURI(gltf, basePath, shader.extras, uri, function() {
```

Comment:
Why create the anonymous function here?  Why not just pass `asyncCallback` (and also just name it `callback`)?